<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>765</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Torgash 2.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/logo.png</normaloff>:/icon/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Network settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="3" column="0">
            <widget class="QLineEdit" name="lineSteamKey"/>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineHttps"/>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="lineSecretKey"/>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineWebsocket"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labelHttps">
             <property name="text">
              <string>https</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="labelWebsocket">
             <property name="text">
              <string>websocket</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="labelSecretKey">
             <property name="text">
              <string>secret key</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="labelSteamKey">
             <property name="text">
              <string>steam key</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Profiles</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QPushButton" name="buttonRemoveProfile">
             <property name="text">
              <string>Remove profile</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="buttonCreateProfile">
             <property name="text">
              <string>Create profile</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Save profile</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="3">
            <widget class="QListWidget" name="listProfiles"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Trades</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QPushButton" name="buttonRunTrades">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Run trades</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="buttonStopTrades">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Stop trades</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="2">
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Log/control</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="2">
            <widget class="QPushButton" name="buttonAddObject">
             <property name="text">
              <string>Add object</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="buttonDeselectAll">
             <property name="text">
              <string>Deselect all</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="buttonSelectAll">
             <property name="text">
              <string>Select all</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="buttonRemoveObject">
             <property name="text">
              <string>Remove object</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QPlainTextEdit" name="plainTextLog">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="plainText">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2">
            <widget class="QPushButton" name="buttonClearLog">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Clear log</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
    <addaction name="actionCreate_2"/>
    <addaction name="actionRemove_2"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuObject">
    <property name="title">
     <string>Object</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove_3"/>
    <addaction name="actionSelect_all"/>
    <addaction name="actionDeselect_all"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="title">
     <string>Log</string>
    </property>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menuProfile"/>
   <addaction name="menuObject"/>
   <addaction name="menuLog"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionObject">
   <property name="text">
    <string>Object</string>
   </property>
  </action>
  <action name="actionCreate_2">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionRemove_2">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemove_3">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionDeselect_all">
   <property name="text">
    <string>Deselect all</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>?</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
